### DISCUSSION TITLE
Can I initialize a repo from subfolders?

### DISCUSSION THREAD

**Discussion by Felipe-Fig** [thebaselab/codeapp#621] [Q&A] [answered]
My CodeApp is installed and it's using the documents folder. But under /documents, let's say I have 2 other folders.

/documents/aaa
/documents/bbb

Is it possible for me to initialize a repository only on /documents/bbb for example?

I'm asking because I've tried and nothing worked so far. Every time I initialize a repo it includes all the subfolders under /documents.

Thanks.

**Comment by bummoblizard** [comment, marked as answer]
Yes. You'll need to set your root directory to aaa or bbb first. Press and hold the folder cell and the option will be revealed. 

I agree that there should be a more intuitive way to do this.
